templates/todo/api/csharp-sql-func/Program.cs

```diff
@@ -0,0 +1,37 @@
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Hosting;
+using Azure.Identity;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.Extensions.Configuration;
+
+namespace SimpleTodo.Api;
+class Program
+{
+    static async Task Main(string[] args)
+    {
+    
+        var host = new HostBuilder()
+                        .ConfigureFunctionsWorkerDefaults()
+                        .ConfigureAppConfiguration(config => 
+                            config.AddAzureKeyVault(new Uri(Environment.GetEnvironmentVariable
```

Comment:
![image](https://user-images.githubusercontent.com/2163001/194171408-c864ae7c-4736-4881-8541-591f8887a9c4.png)

Reply:
I just used ! to get rid of error message but can do a full null check if needed? I didn't see it in the csharp sql template so didn't think it would be necessary here.